packages/mco/components/discovered-application-wizard/enroll-discovered-application.spec.tsx

```diff
@@ -538,3 +548,83 @@ describe('Test replication step', () => {
     expect(screen.getByText('days')).toBeInTheDocument();
   });
 });
+describe('Test review step', () => {
+  beforeEach(() => {
+    testCase += 1;
+    render(<EnrollDiscoveredApplication />);
+    // Select cluster
+    fireEvent.click(screen.getByText('Select cluster'));
+    fireEvent.click(screen.getByText('east-1'));
+
+    // Select namespaces
+    fireEvent.click(screen.getByLabelText('Select row 0'));
+    fireEvent.click(screen.getByLabelText('Select row 1'));
+
+    // Next wizard step
+    fireEvent.click(screen.getByText('Next'));
+
+    if (testCase === 6) {
+      // Select recipe
+      fireEvent.click(screen.getByText('Select a recipe'));
+      fireEvent.click(screen.getByText('mock-recipe-1'));
+
+      // Next wizard step
+      fireEvent.click(screen.getByText('Next'));
+    } else {
+      // ToDo: Resource label selection reivew
+    }
+
+    // Select policy
+    fireEvent.click(screen.getByText('Select a policy'));
+    fireEvent.click(screen.getByText('mock-policy-1'));
+
+    // Next wizard step
+    fireEvent.click(screen.getByText('Next'));
+  });
+  test('Review form test', async () => {
+    // Namespace selection test
+    expect(screen.getAllByText('Namespace').length === 2).toBeTruthy();
+    expect(screen.getByText('Cluster:')).toBeInTheDocument();
+    expect(screen.getByText('east-1')).toBeInTheDocument();
+    expect(screen.getByText('Namespace:')).toBeInTheDocument();
+    expect(screen.getByText('namespace-1, namespace-2')).toBeInTheDocument();
+
+    // Configuration selection
+    expect(screen.getAllByText('Configuration').length === 2).toBeTruthy();
+    expect(screen.getByText('Type:')).toBeInTheDocument();
+    expect(screen.getByText('Recipe')).toBeInTheDocument();
+    expect(screen.getByText('Recipe name:')).toBeInTheDocument();
+    expect(screen.getByText('mock-recipe-1')).toBeInTheDocument();
+    expect(screen.getByText('Recipe namespace:')).toBeInTheDocument();
+    expect(screen.getByText('namespace-1')).toBeInTheDocument();
+
+    // Replication selection
+    expect(screen.getAllByText('Replication').length === 2).toBeTruthy();
+    expect(screen.getByText('Volume replication:')).toBeInTheDocument();
+    expect(
+      screen.getByText(
+        '{{policyName}}, {{replicationType}}, Interval: {{interval}}'
+      )
+    ).toBeInTheDocument();
+    expect(
+      screen.getByText('Kubernetes object replication:')
+    ).toBeInTheDocument();
+    expect(screen.getByText('5 minutes')).toBeInTheDocument();
+
+    // Footer
+    expect(screen.getByText('Save')).toBeInTheDocument();
+    expect(screen.getByText('Back')).toBeEnabled();
+    expect(screen.getByText('Cancel')).toBeInTheDocument();
+
+    // Click save
+    await act(async () => {
+      fireEvent.click(screen.getByText('Save'));
+    });
```

Comment:
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#wrapping-things-in-act-unnecessarily
```suggestion
fireEvent.click(screen.getByText('Save'));
```